src/neuroglancer/util/mouse_drag.ts

```diff
@@ -15,21 +15,25 @@
  */
 
 export type RelativeDragHandler = (event: MouseEvent, deltaX: number, deltaY: number) => void;
-export function startRelativeMouseDrag(initialEvent: MouseEvent, handler: RelativeDragHandler) {
+export type FinishDragHandler = (event: MouseEvent) => void;
```

Comment:
We can use RelativeDragHandler as the type for the finishDragHandler as well, and supply the offsets as well (even if they usually won't be used).